api/src/main/java/org/pmiops/workbench/utils/mappers/CommonMappers.java

```diff
@@ -41,4 +43,14 @@ public static String cdrVersionToId(DbCdrVersion cdrVersion) {
   public static Short dataAccessLevelToStorageEnum(DataAccessLevel dataAccessLevel) {
     return CommonStorageEnums.dataAccessLevelToStorage(dataAccessLevel);
   }
+
+  @Named("cdrVersionToEtag")
+  public static String cdrVersionToEtag(int cdrVersion) {
+    return Etags.fromVersion(cdrVersion);
+  }
+
+  @Named("etagToCdrVersion")
+  public static int etagToCdrVersion(String etag) {
```

Comment:
aside: I could've sworn another mapper already had one of these...

Reply:
I moved them from WorkspaceMapper into CommonMappers